src/components/Counter.component.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class Counter extends Component {
+  state = { count: this.props.initialState }
```

Comment:
Wrap this in a constructor, so like
```
constructor(props) {
     super(props)
     this.state { count: this.props.initialState }
}
```

Reply:
actually, this is more of a matter of style, but it's perfectly fine to leave it as is. i personally prefer to not create constructors if they're not needed